### ISSUE TITLE
YouTube tutorial removed!

### ISSUE THREAD

**Issue by 0verd0se** [ItsCEED/Youtube-Comment-Bot#5]
Please make a new video and upload elsewhere :D

**Comment by Sweely** [comment]
Seconded.

**Comment by raregem009** [comment]
I agree. Please @itsCEED please upload the video somewhere else like mega.nz Thanks

**Comment by Ajeet-Yadav** [comment]
any update?

**Comment by DuckEpik** [comment]
they should post it on ph

**Comment by enthusedcoder** [comment]
Hey guys.  Don't know if this is the guy who created the project, but I found this youtube video that goes over how to use this project. 
https://www.youtube.com/watch?v=6FNWI9z_5J4

**Comment by ItsCEED** [comment]
@enthusedcoder this has nothing to do with this.

also the code was outdated even when inuploaded this.

that being said, its not worth even trying to run this
